docs/features/sessions/changing-instance-type.md

```diff
@@ -0,0 +1,30 @@
+---
+description: Run (and sweep) any private or public Github repository.
+---
+import Note from "@site/src/components/Note";
+
+# Change Instance Type of a Session
+
+```text
+grid session change-instance-type [--spot , --on_demand] SESSION_NAME g4dn.xlarge
+```
+
+Changing Sessison instance type allows you to upgrade or downgrade the compute capacity of the Session while keeping all of your work in progress untouched.
+
+![](/images/sessions/change-instance-type.gif)
+
+
+The equivalent via the CLI command:
+
+![](/images/sessions/change-instance-type.png)
+
+You should PAUSE the session to change the instance type or if you change the instance type when the session is running, you should PAUSE and RESUME the session for the changes to take effect.
+
+The following scenarios are supported: 
+1. Changing from a CPU instance to another CPU instance
+2. Changing from a GPU instance to another GPU instance
+3. Changing from a GPU instance to a CPU instance
+4. Changing an On-Demand instance to a Spot Instance
+5. Changing from a Spot Instance to an On-Demand Instance
+
```

Comment:
@sunitaprakash Robert L mentioned that the only scenario that is currently supported is CPU to CPU. Can you update this to reflect that and mention that we will expand to support upscaling? 

You should also make sure the examples we share show CPU to CPU since that is the only one that works right now.